myproject/myapp/serializers.py

```diff
@@ -0,0 +1,17 @@
+# myapp/serializers.py
+from rest_framework import serializers
+from .models import CustomUser
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CustomUser
+        fields = ['id', 'username', 'email', 'password']
```

Comment:
use `email_address` instead. @devo2104

Reply:
Hey Mudit, I have done the changes. Resolving this conversation now.